app/services/scholars_archive/workflow/deposit_received_notification.rb

```diff
@@ -12,9 +12,18 @@ def subject
 
       # rubocop:disable Metrics/AbcSize
       # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Lint/Void
       def message
         if !SolrDocument.find(work_id)['resource_type_tesim'].include?('Dataset')
           "ScholarsArchive@OSU has received your deposit: #{title} (#{link_to work_id, citeable_url}). Your item is under review by repository administrators. You will be notified if your deposit requires additional changes and/or when your deposit is live in the repository. \n\n #{comment}"
+          "<br /> If the submitted dataset includes human subjects data, you will need to facilitate the following information to the data curator:\n"
```

Comment:
Is this the text from Clara? There's a grammar error in `you will need to facilitate the following information to the data curator`

Reply:
It is from Clara, I try to fix it and would this work? `you will need to give the following information for the data curator`